oci/layout/oci_archive_dest.go

```diff
@@ -0,0 +1,126 @@
+package layout
```

Comment:
Only as long as the code keeps relying on the internals of `ociReference` etc., which should not be necessary if the rest of the code converts to use temporary directories.